src/main/fc/rc_adjustments.c

```diff
@@ -667,7 +667,7 @@ static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, ui
         static uint8_t lastPosition = 255;
 
         if (position != lastPosition) {
-            int newMultiplier = scaleRange(position, 5, 95, 5, 200);
+            int newMultiplier = scaleRange(position, 5, 90, 5, 200);
```

Comment:
What is the intent here? position is IMO 0..100 so it is cleaner to use it as input range (and adjust output range as desired). With `5, 90`, you should constrain position

Reply:
Trying out some options.  Noticing that it is not using the full range of the slider (0.5 to 2.0) when moving the full rnage of the POT (1000 to 2000).  I keep pushing to use the auto-builder for flashing/testing.  

I'm going to push some new code to test.  Ideally, when activated, it will use the delta change in the POT, not the absolute position.  That way, when you activate with the switch, your POT can be in any position and as you spin it, the slider will move up/down, but be bounded by the range.  If at the end of the POT range for some reason, or didn't have centered when activated, you can switch off, center the POT, and switch back on to have more range in the POT to adjust the slider position.

Reply:
I tried to implement similar adjustment long time ago, but I did not find satisfactory implementation. I ended up using one pot for each adjustment.

Reply:
@ledvinap figured it out.  Works great so far.  

Wherever the POT is set, when you activate the RC Adjustment, that is the center point for the adjustment (up or down).  This way, you don't have to be so worried about the POT being centered (absolute position) before activating the In-Flight Adjustment.

Will test fly it.